srcpkgs/oblibs/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'oblibs'
+pkgname=oblibs
+version=0.0.1.1
+revision=1
+wrksrc="${pkgname}-v${version}"
+build_style=configure
+configure_args="--prefix=/usr --libdir=/usr/lib"
+makedepends="skalibs-devel"
+short_desc="Obarun library"
+maintainer="mobinmob <mobinmob@disroot.org>"
+license="ISC"
+homepage="https://framagit.org/Obarun/oblibs"
+distfiles="${homepage}/-/archive/v${version}/${pkgname}-v${version}.tar.bz2"
+checksum=fbb0eaf9223805124f76c086a2d1a444649d077ef86bc05762776b8c0cfad1b3
+
+post_install() {
+	vlicense COPYING
+}
+
+oblibs-devel_package() {
+	depends="${makedepends} ${sourcepkg}-${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		vmove "usr/lib/*.a"
+		vmove usr/include/oblibs
+		vmove "usr/lib/*.so"
+	}
+}
```

Comment:
a newline character is missing at the end of this line.

Reply:
Fixed.